vkcs/resource_vkcs_images_image.go

```diff
@@ -231,6 +231,9 @@ func resourceImagesImageCreate(ctx context.Context, d *schema.ResourceData, meta
 
 	properties := d.Get("properties").(map[string]interface{})
 	imageProperties := resourceImagesImageExpandProperties(properties)
+	if !resourceImagesImageNeedsDefaultStore(imageClient.Endpoint) {
+		imageProperties["store"] = "s3"
```

Comment:
Maybe move `"s3"` to well-named constants?

Reply:
Moved